renderer/main.js

```diff
@@ -1213,11 +1213,7 @@ function openTorrentContextMenu (infoHash) {
 }
 
 function getTorrentPath (torrentSummary) {
-  var itemPath = path.join(torrentSummary.path, torrentSummary.files[0].path)
-  if (torrentSummary.files.length > 1) {
-    itemPath = path.dirname(itemPath)
-  }
-  return itemPath
+  return path.join(torrentSummary.path, torrentSummary.name)
```

Comment:
I don't think the bug is about single file torrents.
In a multi file torrent, when the first file `files[0]` is within a subfolder, the resolved path will be that of the subfolder and not root torrent folder.